module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,9 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> words){
+    words.sort();
```

Comment:
You need to add more test cases in test :) That was tricky part. 
Here you sort only alphabetically, but not  by length.

You need to use std::sort() with special predicat, to check length and alphabetical order.

Reply:
As I said to WBigaj above. It is related to STL_1 homework and it is fixed on branch related with 1 homework.